cypress/pages/appSettingsPage.ts

```diff
@@ -0,0 +1,12 @@
+const AppSettingsPage = {
```

Comment:
We have an `settingsflow.spec.ts` that this test can fall under

Reply:
I see these page objects as living separate from the test files. I want to make this reusable across multiple tests. so for example, `settings.flow.spec.ts` could have tests of user journeys relates to the app settings and `dashboard.spec.ts` can have tests related to actions on the dashboard, but they both need to perform some action on the App settings page, so thats why i've kept `AppSettingsPage` a separate thing

Reply:
Thanks for the call around this this morning @ghernandez345